src/sections/MySocial.jsx

```
@@ -0,0 +1,38 @@
+    <section id="mysocial" className="flex-center section-padding">
+      <div className="w-full h-full md:px-10 px-5">
+        <TitleHeader title="" sub="⭐️ Follow me on social media" />
+
+        <div className="lg:columns-3 md:columns-2 columns-1 mt-16">
+          {socialMedia.map((socialMedia, index) => (
+            <a
+              href={socialMedia.link}
+              className="text-blue-500 hover:underline cursor-pointer"
+              target="_blank"
+              rel="noopener noreferrer"
+            >
```

Biome (2.4.4)

[error] Missing key property for this element in iterable.

(lint/correctness/useJsxKeyInIterable)